CliTest.test_cli_cmd_loop_quit test fails on macOS

Environment:
- Freshly installed macOS Sierra (10.12) (in a VM)
- python3.6

```
___________________________________________________________________________________ CliTest.test_cli_cmd_loop_quit ____________________________________________________________________________________

self = <synapse.tests.test_lib_cli.CliTest testMethod=test_cli_cmd_loop_quit>

    def test_cli_cmd_loop_quit(self):

        @mock.patch('synapse.lib.cli.get_input', return_value='quit')
        def _innertest(testcase, *args, **kwargs):
            with s_cli.Cli(None) as cli:
                cli.runCmdLoop()
                testcase.eq(cli.isfini, True)

>       _innertest()

synapse/tests/test_lib_cli.py:114:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/usr/local/Cellar/python3/3.6.3/Frameworks/Python.framework/Versions/3.6/lib/python3.6/unittest/mock.py:1179: in patched
    return func(*args, **keywargs)
synapse/tests/test_lib_cli.py:111: in _innertest
    cli.runCmdLoop()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <synapse.lib.cli.Cli object at 0x124791f28>

    def runCmdLoop(self):
        '''
            Run commands from stdin until close or fini().
            '''
        import readline
>       readline.read_init_file()
E       FileNotFoundError: [Errno 2] No such file or directory

synapse/lib/cli.py:245: FileNotFoundError
```

According to [this website](https://www.gnu.org/software/bash/manual/html_node/Readline-Init-File.html), the file it's looking for is either `~/.inputrc` or `/etc/inputrc` as a fallback. On my fresh OSX VM, neither of those files exists.

--- a/synapse/lib/cli.py
+++ b/synapse/lib/cli.py
@@ -277,7 +277,15 @@ class Cli(s_eventbus.EventBus):
         Run commands from a user in an interactive fashion until fini() or EOFError is raised.
         '''
         import readline
-        readline.read_init_file()
+        try:
+            readline.read_init_file()
+        except OSError:
+            # from cpython 3.6 site.py:
+            # An OSError here could have many causes, but the most likely one
+            # is that there's no .inputrc file (or .editrc file in the case of
+            # Mac OS X + libedit) in the expected location.  In that case, we
+            # want to ignore the exception.
+            pass
 
         while not self.isfini:
 

